lib/web_of_science/process_records.rb

```diff
@@ -90,6 +87,17 @@ def create_publication(record, wssr)
         message: record.uid,
         publication: publication
       }))
+      send_recruitemail(wssr, publication)
+    end
+
+    # @param [WebOfScienceSourceRecord] WebOfScienceSourceRecord
+    # @param [Publication] Publication
+    # @return [Boolean] RECRUIT_EMAIL job completed?
+    def send_recruitemail(wssr, publication)
+      contact_emails = WebOfScience::SendRecruitemail.scrape_emails(wssr, publication)
```

Comment:
I think we should create a Job for fetching the email addresses, and then setting it as started. 

If the method raises an error, then set the job to "error" or to set it "completed" when it was completed.

Something along the lines of this

```
def fetch_email
  job = Job.create(Job::FETCH_EMAILS.merge({ status: Job::STARTED[:name]}))
  contact_emails = WebOfScience::SendRecruitemail.scrape_emails(wssr, publication)
  job.completed({ message: 'Fetched author emails addresses.' })
  contact_emails
rescue => e
  job.error({ message: e.message })
  raise e
end
```